_posts/2024-10-06-update.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Week Ending October 06, 2024
+date: 2024-10-06 22:00:00 -0000
```

Comment:
Did you mean:
```suggestion
date: 2024-10-06T22:00:00-0000
```
?

Reply:
The `T` is not needed. Confirmed with the last issue:

https://github.com/kubernetes-sigs/lwkd/blob/c60b1d1ed2529bffbfd6067bc83e3f1440cc8df0/_posts/2024-09-29-update.md?plain=1#L4